CHANGELOG.rst

```diff
@@ -1,5 +1,61 @@
 Changelog
 =========
+1.13 (2020-12-22)
+-----------------
+MLflow 1.13 includes several major features and improvements:
+
+Features:
+
+New fluent APIs for logging in-memory objects as artifacts:
+
+- Add `mlflow.log_text` (#3678, @harupy)
```

Comment:
@harupy as mentioned in https://github.com/mlflow/mlflow/pull/3790, this might introduce an API inconsistency:

> I am not sure why the file path argument [in log_text() and friends] was named `artifact_file`, as my impression is that `path` and `artifact_path` are far more common in the source code (1400+ vs 80+ matches). I think the argument `artifact_file` in the Python API should be renamed to `artifact_path` or `path` for consistency with `mlflow.log_artifacts()` / `list_artifacts()` and the rest of the API. I think this is still possible without breaking the released API, as this change was not yet released. Might also be it was deliberately named `artifact_file`, I don't know. 

Sorry for bothering if this was intentional or not a concern.

Reply:
Thanks for the commemt. `artifact_path` represents a directory, not a file path. That’s why we chose `artifact_file`.

Reply:
I see, thanks for the explanation.

Reply:
We also considered the following design:

```python
def log_xxx(obj, filename, artifact_path):
    ...
```

This would enforce users to specify two arguments when loggigng an artifact in a subdirectory, which seems a bit inconvenient.